packages/analytics-browser/src/browser-client.ts

```diff
@@ -65,6 +71,8 @@ export class AmplitudeBrowser extends AmplitudeCore<BrowserConfig> {
     await this.add(new IdentityEventSender());
     await this.add(new Destination());
 
+    this.initializing = false;
```

Comment:
Safe to allow another init after plugin installation. The next operations are sending track calls.